### ISSUE TITLE
example with css

### ISSUE THREAD

**Issue by joprice** [ProdriveTechnologies/bazel-pandoc#12]
Is there a way to add css to an html output?

**Comment by patbro** [comment]
Hi there! The Pandoc documentation mentions: _"When HTML is used as an intermediate format, the output can be styled using --css."_ So, please feel free to contribute by including this CLI argument in `pandoc.bzl`.

**Comment by joprice** [comment]
I tried that and couldn't get it to work. It silently builds only the html, so I was thinking that the rule has to add that as an extra source file or something along those lines.

**Comment by patbro** [comment]
Yes! That's exactly what needs to be added to `pandoc.bzl` and of course the build file you are building.

I tried putting together some sample code which builds with bazel-pandoc, but Pandoc doesn't seem to pick-up the stylesheet itself. I will try to run Pandoc with `--css` and see whether that gives the same result later this week.

**Comment by patbro** [comment]
Quick update! I put an initial concept together to support CSS for HTML documents in PR #13, please have a look.
